src/Bridge/Doctrine/Orm/Filter/BooleanFilter.php

```diff
@@ -0,0 +1,143 @@
+     *
+     * For each property passed, if the resource does not have such property or if the order value is different from
+     * 1, 0, true, false, on, off (case insensitive) the property is ignored.
+     */
+    public function apply(QueryBuilder $queryBuilder, string $resourceClass, string $operationName = null)
+    {
+        $request = $this->requestStack->getCurrentRequest();
+        if (null === $request) {
+            return;
+        }
+
+        $properties = $this->extractProperties($request);
+
+        foreach ($properties as $property => $boolean) {
+            if (!$this->isPropertyEnabled($property) || !$this->isPropertyMapped($property, $resourceClass)) {
+                continue;
+            }
+
+            if (empty($boolean) && isset($this->properties[$property])) {
+                $boolean = $this->properties[$property];
+            }
+            if (is_null(filter_var($boolean, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE)) && !empty($boolean)) {
+                continue;
+            }
+
+            $alias = 'o';
+            $field = $property;
+
+            if ($this->isPropertyNested($property)) {
+                $propertyParts = $this->splitPropertyParts($property);
+
+                $parentAlias = $alias;
+
+                foreach ($propertyParts['associations'] as $association) {
+                    $alias = QueryNameGenerator::generateJoinAlias($association);
+                    $queryBuilder->leftJoin(sprintf('%s.%s', $parentAlias, $association), $alias);
+                    $parentAlias = $alias;
+                }
+
+                $field = $propertyParts['field'];
+            }
+            $valueParameter = QueryNameGenerator::generateParameterName(sprintf('%s_%s', $field, $boolean));
+
+            $queryBuilder
+                ->andWhere(sprintf('%s.%s = :%s', $alias, $field, $valueParameter))
+                ->setParameter($valueParameter, filter_var($boolean, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE));
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription(string $resourceClass) : array
+    {
+        $description = [];
+
+        $properties = $this->properties;
+        if (null === $properties) {
+            $properties = array_fill_keys($this->getClassMetadata($resourceClass)->getFieldNames(), null);
+        }
+
+        foreach ($properties as $property => $boolean) {
+            if (!$this->isPropertyMapped($property, $resourceClass) || !$this->isBooleanField($property, $resourceClass)) {
+                continue;
+            }
+
+            $description[$property] = [
+                'property' => $property,
+                'type' => 'boolean',
+                'required' => false,
+            ];
+        }
+
+        return $description;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function extractProperties(Request $request) : array
```

Comment:
`extractProperties` is already in `AbstractFilter`, and it also handles fixing of nested properties.